Update create-openapi-repo README (#64)
Redocly/create-openapi-repo: README.md

# create-openapi-repo

A tool for generating multi-file OpenAPI definitions.

[![NPM version][npm-image]][npm-url] [![Dependency Status][daviddm-image]][daviddm-url]

<p align="center">
<img src="./logo.png" width="500px"/>
</p>

Need to write or contribute to an OpenAPI definition? `create-openapi-repo` can help with that!

## Features

 - OpenAPI 3.0 support
 - Split an existing OpenAPI definition into multiple files
 - Bundle a multi-file definition into a single file
 - Validate your OpenAPI definition using our free [openapi-cli tool](https://github.com/redocly/openapi-cli)
 - Automate deployment of your API reference docs using CI/CD workflows
 - Maintain code samples as separate files
 - Live editing in your editor of choice :heart_eyes:

> ## Examples

Reviewer: not sure about the last two examples - one is archived and the other hasn't been touched in seven years

- [Rebilly](https://github.com/Rebilly/RebillyAPI)
- [Thingful](https://github.com/thingful/openapi-spec)

> - [Fitbit Plus](https://github.com/TwineHealth/TwineDeveloperDocs)

Reviewer: ```suggestion
```

## Prerequisites

> Before you begin, make sure you’ve installed the following prerequisites:

Reviewer: ```suggestion
Before you begin, make sure you have the following prerequisites:
```

```
    ├── .redocly.yaml
    ├── LICENSE
    ├── README.md
    ├── docs
    │   ├── favicon.png
    │   └── index.html
    ├── openapi
    │   ├── README.md
    │   ├── code_samples
    │   │   ├── C#
    │   │   │   └── echo
    │   │   │       └── post.cs
    │   │   ├── PHP
    │   │   │   └── echo
    │   │   │       └── post.php
    │   │   └── README.md
    │   ├── components
    │   │   └── README.md
    │   └── paths
    │       └── README.md
    └── package.json
```

 - `.redocly.yaml`: Configuration file for defining settings for various Redocly tools, including the lint tool and reference docs engine.
 - `openapi`: Top-level folder that contains your OpenAPI definition, `openapi.yaml` entrypoint file, and sub-folders for `paths`, `components`, and `code_samples`.
 - `code_samples`: Folder for organizing code samples into sub-folders, such as C# and PHP.
 - `components`: Folder for organizing reusable components into sub-folders, such as `schema` and `response` objects.
 - `paths`: Folder for organizing path definitions. Each path should be referenced from the `openapi.yaml` entrypoint file.

## Commands

> The generated repository installs a dependency for our `openapi-cli` tool which supports the following commands:

Reviewer: Isn't it redocly-cli now?

> The generated repository installs a dependency for our `openapi-cli` tool which supports the following commands:

Reviewer: ```suggestion
The generated repository installs a dependency for our `redocly-cli` tool which supports the following commands:
```
